pkg/components/rook-ceph/component.go

```diff
@@ -37,12 +37,20 @@ type component struct {
 	MetadataDevice string              `hcl:"metadata_device,optional"`
 	Tolerations    []util.Toleration   `hcl:"toleration,block"`
 	TolerationsRaw string
+	StorageClass   *StorageClass `hcl:"storage_class,block"`
```

Comment:
If we set it by default, maybe make it non-pointer type?

Reply:
The pointer is needed to make it optional. It fails otherwise:

```console
$ lokoctl component render-manifest rook-ceph
ERRO[0000] packet-cluster.lokocfg:101,23-23: Missing storage_class block; A storage_class block is required.  component=rook-ceph
FATA[0000] packet-cluster.lokocfg:101,23-23: Missing storage_class block; A storage_class block is required.  args="[rook-ceph]" command="lokoctl component render-manifest"
```